requirements/base.txt

```diff
@@ -2,30 +2,43 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile --allow-unsafe --output-file=requirements/base.txt requirements/base.in
+#    pip-compile --max-rounds=100 --no-emit-index-url requirements/base.in
 #
 attrs==20.3.0             # via -r requirements/base.in
 awesome-slugify==1.6.5    # via -r requirements/base.in
-base58==2.0.1             # via -r requirements/base.in
+base58==2.0.1             # via -r requirements/base.in, scalecodec, substrate-interface
 cached-property==1.5.2    # via -r requirements/base.in
+certifi==2020.12.5        # via requests, substrate-interface
+chardet==4.0.0            # via requests
 click-default-group==1.2.2  # via -r requirements/base.in
 click==7.1.2              # via -r requirements/base.in, click-default-group, pip-tools, rq
 colorlog==4.6.2           # via -r requirements/base.in
+idna==2.10                # via requests, substrate-interface
+more-itertools==8.6.0     # via scalecodec
 numpy==1.19.4             # via -r requirements/base.in
 pendulum==2.1.2           # via -r requirements/base.in
 pip-tools==5.4.0          # via -r requirements/base.in
 plotly==4.14.1            # via -r requirements/base.in
+py-bip39-bindings==0.1.6  # via substrate-interface
+py-ed25519-bindings==0.1.2  # via substrate-interface
+py-sr25519-bindings==0.1.2  # via substrate-interface
 python-dateutil==2.8.1    # via pendulum
 python-rex==0.4           # via -r requirements/base.in
 pytz==2020.5              # via -r requirements/base.in
 pytzdata==2020.1          # via pendulum
 redis==3.5.3              # via -r requirements/base.in, rq
 regex==2020.11.13         # via awesome-slugify
+requests==2.25.1          # via scalecodec, substrate-interface
 retrying==1.3.3           # via plotly
 rq==1.7.0                 # via -r requirements/base.in
-six==1.15.0               # via pip-tools, plotly, python-dateutil, python-rex, retrying
+scalecodec==0.10.49       # via substrate-interface
+six==1.15.0               # via pip-tools, plotly, python-dateutil, python-rex, retrying, websocket-client
 sortedcontainers==2.3.0   # via -r requirements/base.in
+substrate-interface==0.11.10  # via -r requirements/base.in
 unidecode==0.4.21         # via awesome-slugify
+urllib3==1.26.2           # via requests
+websocket-client==0.57.0  # via substrate-interface
+xxhash==2.0.0             # via substrate-interface
 
 # The following packages are considered to be unsafe in a requirements file:
-pip==20.3.1               # via -r requirements/base.in, pip-tools
+# pip
```

Comment:
Did `fab compile` produce that ? If yes no problem there, if you did this manually I think that it has to be reverted.

Reply:
Hey Spyros, I didn't do it manually